mailu/values.yaml

```diff
@@ -108,6 +108,7 @@ certmanager:
 ingress:
   externalIngress: true
   tlsFlavor: cert
+  className: ""
```

Comment:
Not 100% sure but I think if you default it to empty string, the result is different from omitting it completely. So this would result in not using any ingress (not even the default) - right?

Reply:
This is correct! However please note that in the template, the `ingressClassName` attribute is conditioned to `.Values.className` being truthy:
```
{{- with .Values.ingress.className }}
```

So if `.Values.className` is empty, `ingressClassName` will be absent from the ingress manifest.

Reply:
Ok, that's what I was unsure, if the "with" will then trigger or not.